web/src/flow/stages/authenticator_validate/AuthenticatorValidateStage.ts

```diff
@@ -33,6 +64,10 @@ export class AuthenticatorValidateStage
     >
     implements StageHost
 {
+    static get styles(): CSSResult[] {
```

Comment:
I extracted the customCSS because it was big enough it distracted from the main purpose of the class, so this makes the entirety a bit more readable.

Also, there's an order to Lit components that we've adopted: Styles, Properties, States, other class fields, constructor, connectedCallback, disconnectedCallback, event handlers, and then the lifecycle methods from `performUpdate()` through `render()` and finally `updated()`.  (Obviously, don't include any that you don't override!) Yes, this means that lifecycle methods that run after `render()` appear after your `render()` method in the class.  Helper methods (sub-renderers, formatters, etc) go _above_ the first lifecycle method in which they're used, in the order in which they appear in the lifecycle method.  We're not perfect about this, but we are getting better.